lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_bump_version_final_release.rb

```diff
@@ -23,11 +32,21 @@ def self.run(params)
       #####################################################
 
       def self.description
-        'Bumps the version of the app for a new beta'
+        'Bumps the version of the app for a new beta. Depends on a gradle task to update the keys in a version.properties file.'
       end
 
       def self.details
-        'Bumps the version of the app for a new beta'
+        'Bumps the version of the app for a new beta. Depends on a gradle task to update the keys in a version.properties file.'
+      end
+
+      def self.available_options
+        # Define all options your action supports.
+        [
+          FastlaneCore::ConfigItem.new(key: :app,
+                                       env_name: 'PROJECT_NAME',
+                                       description: 'The name of the app to get the release version for',
+                                       is_string: true), # true: verifies the input is a string, false: every kind of value
```

Comment:
This is legacy api of `ConfigItem` constructor. The more modern version is:
```suggestion
                                       type: String),
```

This is not a blocker and can be fixed in a subsequent PR; but could be nice to take the occasion to fix while you're at it; this suggestion applies in every places and actions where you added this of course.

Reply:
Ahhh I think I'd rather tackle it in another PR, opened https://github.com/wordpress-mobile/release-toolkit/issues/278 to track it.